docs/content/get-started/build-workers.md

```diff
@@ -4,11 +4,11 @@ As you will be launching a cloud deployment, you need to build out the code exec
 
 In the Unity Editor, you first need to make sure Burst compilation is **disabled** from **Jobs** > **Enable Burst Compilation**. Then you can build your workers from the SpatialOS menu by clicking **Build for cloud** > **All workers**.
 
-<%(Callout type="info" message="Building workers for the first time may take while. Why not make yourself a cup of tea while you wait?")%>
+<%(Callout type="info" message="Building workers for the first time may take while (about 10 minutes). Why not make yourself a cup of tea while you wait? Or check out [our Youtube channel](https://www.youtube.com/channel/UC7BE8B2yUeQxPvZytk47NYw/videos)? Or both!")%>
```

Comment:
We are trying to avoid covering all the docs in multi-coloured messages .
  ```suggestion
>Building workers for the first time may take while (about 10 minutes). Why not make yourself a cup of tea or check out [our Youtube channel](https://www.youtube.com/channel/UC7BE8B2yUeQxPvZytk47NYw/videos) while you wait?
```

Reply:
Note to self - Added in new PR.